autosklearn/estimators.py

```diff
@@ -475,6 +476,7 @@ def __init__(
         self.get_trials_callback = get_trials_callback
         self.dataset_compression = dataset_compression
         self.allow_string_features = allow_string_features
+        self.disable_progress_bar = disable_progress_bar
```

Comment:
Also take a look at putting this into `autosklearn.experimental.AutoSklearn2Classifier`